controllers/nutanixmachine_controller.go

```diff
@@ -362,7 +362,7 @@ func (r *NutanixMachineReconciler) reconcileDelete(rctx *nctx.MachineContext) (r
 		}
 		log.V(1).Info(fmt.Sprintf("no running tasks anymore... Initiating delete for VM %s with UUID %s", vmName, vmUUID))
 	} else {
-		log.V(1).Info(fmt.Sprintf("no task UUID found on VM %s. Starting delete.", *vm.Spec.Name))
+		log.V(1).Info(fmt.Sprintf("no task UUID found on VM %s. Starting delete.", vmName))
```

Comment:
Do detach VG tasks get associated with the VM? If so then we don't need to wait for tasks at all if we've fired off the VG detachments, do we?

Reply:
it's a VG api call and not a VM api call. So, I'd be somewhat doubtful. I agree we don't need to wait for tasks for now. So, let's remove them as agreed offline.